network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -9,6 +15,102 @@
 from networkapi.wagtailpages.pagemodels.mixin.foundation_metadata import (
     FoundationMetadataPageMixin
 )
+from networkapi.buyersguide.pagemodels.product_update import Update
+
+if settings.USE_CLOUDINARY:
+    image_field = FieldPanel('cloudinary_image')
+else:
+    image_field = ImageChooserPanel('image')
+
+
+class ProductPageCategory(Orderable):
+    product = ParentalKey(
+        'wagtailpages.ProductPage',
+        related_name='product_categories',
+        on_delete=models.CASCADE
+    )
+    category = models.ForeignKey(
+        'buyersguide.BuyersGuideProductCategory',
+        related_name='+',
+        blank=False,
+        null=True,
+        on_delete=models.SET_NULL,
+    )
+
+    panels = [
+        SnippetChooserPanel('category')
+    ]
+
+    def __str__(self):
+        return self.category.name
+
+    class Meta:
+        verbose_name = "Product Page Category"
```

Comment:
The "Page" part is usually not super great, especially as in this case it'd be more like a `ProductPageCategoryPage`

```suggestion
        verbose_name = "Product Category"
```